.github/workflows/Readme.md

```diff
@@ -0,0 +1,76 @@
+Tips for shorter build times
+----------------------------
+
+### Runner availability ###
+
+Currently, the biggest bottleneck when optimizing workflows is the availability
+of Windows and macOS runners. Therefore, anything that reduces the time spent in
+Windows or macOS jobs will have a positive impact on the time waiting for
+runners to become available. You can see a breakdown of runner usage in the
+Github Actions tab
+([example][usage]).
+
+### Windows is slow ###
+
+Performing git operations and compilation are both slow on Windows. This can
+easily mean that a Windows job takes twice as long as a Linux job. Therefore it
+makes sense to use a Windows runner only for testing Windows compatibility, and
+nothing else. Testing compatibility with other versions of Nim, code coverage
+analysis, etc. are therefore better performed on a Linux runner.
+
+### Parallelization ###
+
+Breaking up a long build job into several jobs that you run in parallel can have
```

Comment:
I found this: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits
Not sure if that's our actual limit (no idea what our plan is), but it seems that the limitation is 20 concurrent jobs for the organization (or 5 for MacOS)

Reply:
Thanks! I added a link to the usage limits in the readme.